src/k8s-engine/wsl.ts

```diff
@@ -1104,6 +1104,33 @@ export default class WSLBackend extends events.EventEmitter implements K8s.Kuber
     await this.execCommand('/usr/local/bin/wsl-service', service, 'start');
   }
 
+  /**
+   * Verify that the given command runs successfully
+   * @param command
+   * @protected
+   */
+  protected async verifyReady(...command: string[]) {
+    const startTime = Date.now();
+    const maxWaitTime = 60_000;
+    const waitTime = 200;
+
+    while (true) {
+      const currentTime = Date.now();
+
+      if ((currentTime - startTime) > maxWaitTime) {
+        console.log(`Waited more than ${ maxWaitTime / 1000 } secs for ${ command.join(' ') } to succeed. Giving up.`);
+        break;
+      }
+      try {
+        await this.execCommand(...command);
```

Comment:
```suggestion
        await this.execCommand({ expectFailure: true }, ...command);
```
This prevents us from logging the failure (because it's expected); it'll still throw, though.

Reply:
We're logging the command output here; would that be excessive?

Reply:
I wasn't seeing any output from this command when it failed, just the exception, but it doesn't matter -- it's normally a transient condition.